lib/common.php

```diff
@@ -295,3 +295,20 @@ function wc_admin_is_feature_enabled( $feature ) {
 	$features = wc_admin_get_feature_config();
 	return isset( $features[ $feature ] ) && true === $features[ $feature ];
 }
+
+/**
+ * Format a number using the decimal and thousands separator settings in WooCommerce.
+ *
+ * @param int $number Number to be formatted.
```

Comment:
Does this support floats as well?

Reply:
You probably wouldn't want to pass a float here since it removes all decimals, but it does support them.  Updated this comment to `mixed`.